libs/watt/src/lib/components/expandable-card/watt-expandable-card.component.html

```diff
@@ -14,7 +14,7 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<mat-expansion-panel [expanded]="expanded">
+<mat-expansion-panel [expanded]="expanded" class="watt-expandable-card">
```

Comment:
question: doesn't this still duplicate the `watt-expandable-card` class - once here and once in `WattExpandableCardComponent` class?

suggestion: I was thinking of something like

```html
<mat-expansion-panel [expanded]="expanded" class="watt-expandable-card watt-{{ variant }}">
```

then remove the `HostBinding` from the class, and adjust the CSS file to this:

```css
.watt-expandable-card.mat-expansion-panel {
  ...
  &.watt-solid {
    border: 1px solid var(--watt-color-neutral-grey-300);
    box-shadow: none;
  }
}
```